CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   * option --update has been added - sets percentage of update operations in bench space.
   * option --fill" has been added - sets number of records to pre-fill the space.
 
+- An ability to choose tarantool version to be installed in a docker image:
+  * command line option --tarantool-version
+  * tarantool.txt config file with TARANTOOL option
```

Comment:
Dependencies are a common term for rpm/deb, but not for docker image. Also I think we may have more confusing situations like:
cartridge pack --deps "tarantool>=2.8.2" --deps "libz>1.0" docker
Must it install specific version of libz on docker image or silently ignore one of the --deps options? This looks odd for me.
If we ignore some lib names in deps/depsfile for docker pack, I can mistype "tarantool" and this will not lead to error.

Reply:
Can't we just pass all those values to apt-get/yum (if there is tarantool here, enable tarantool and modules repository beforehand)?

Reply:
We can do everything :)
I think this deserves a separate feature request for installing packages specified in --deps to docker images. Because this looks like an overkill for the task to just specify tarantool version.

Reply:
It seems, it is not so simple. As I see, we can't just do something like `yum install foo>=1.2`. In theory, we can generate an RPM spec with all needed dependencies (`BuildRequires`) and run `rpmbuild`, but it looks complicated and would bloat the resulting image with the RPM build tools (or would require cleaning it up -- more code).

So, okay, let's introduce a separate option for choosing tarantool version.